cmd/plume/upload.go

```diff
@@ -95,17 +110,76 @@ func runUpload(args []string) int {
 		return 1
 	}
 
-	fmt.Printf("Writing %v to %v\n", imageName, bucket)
+	fmt.Printf("Writing %v to gs://%v ...\n", imageNameGS, bucket)
+	fmt.Printf("(Sometimes this takes a few mintues)\n")
+
+	/*
```

Comment:
Doh! Yes, this is left over debugging. I was looking at the wrong line. I thought you were talking about the taking a few minute line for some reason.